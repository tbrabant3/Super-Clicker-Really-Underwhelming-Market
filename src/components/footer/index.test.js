import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './index';

test('Should Render Footer', () => {
	const component = renderer.create(<Footer />);
	const tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
