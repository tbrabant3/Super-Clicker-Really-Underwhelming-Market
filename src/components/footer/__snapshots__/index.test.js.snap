// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Render Footer 1`] = `
<div>
  footer
</div>
`;
